contrib/qubes/doc/whonix.md

```diff
@@ -0,0 +1,30 @@
+# Whonix support
+
+The qubes-fwupd uses the sys-whonix VM as the update VM to handle downloading
+updates and metadata via Tor. The tests detect if sys-whonix is running, but
+do not check if you are connected with Tor. So before running the test make sure
+that sys-whonix has access to the network.
+
+## Refresh
+
+```
+$ sudo qubes-fwupdmgr refresh --whonix
+```
+
+[![asciicast](https://asciinema.org/a/5zJhIZeATwx9OYVOELoK69ZMo.svg)](https://asciinema.org/a/5zJhIZeATwx9OYVOELoK69ZMo)
+
+## Update
+
+```
+$ sudo qubes-fwupdmgr update --whonix
+```
+
+[![asciicast](https://asciinema.org/a/fsqv5Q3Bzi79LoCqoQB2RKts0.svg)](https://asciinema.org/a/fsqv5Q3Bzi79LoCqoQB2RKts0)
```

Comment:
I do wonder if we want to include these "videos" in the source tree; is asciinema.org going to exist in 5 years?

Reply:
The oldest video on our blogpost is 2 years old. 

> Is asciinema.org going to exist in 5 years?

That's a good question. I can remove these links if we don't want to carry about them in the future.

Reply:
Maybe a better place to put them is in a blog post or wiki or somewhere easier to spin than a source tree.